Wabbajack.Lib/ABatchProcessor.cs

```diff
@@ -47,13 +48,18 @@ public abstract class ABatchProcessor : IBatchProcessor
 
         private readonly CompositeDisposable _subs = new CompositeDisposable();
 
-        protected void ConfigureProcessor(int steps, int threads = 0)
+        // WorkQueue settings
+        public BehaviorSubject<bool> ManualCoreLimit = new BehaviorSubject<bool>(true);
+        public BehaviorSubject<byte> MaxCores = new BehaviorSubject<byte>(byte.MaxValue);
+        public BehaviorSubject<double> TargetUsagePercent = new BehaviorSubject<double>(1.0d);
```

Comment:
Not too happy on the presence of these settings here, but not sure where they should go exactly right now.  Maybe moving them to WorkQueue itself could be a better spot for them.